libc-test/build.rs

```diff
@@ -2730,7 +2730,6 @@ fn test_emscripten(target: &str) {
     cfg.define("_GNU_SOURCE", None); // FIXME: ??
 
     headers! { cfg:
-               "aio.h",
```

Comment:
Most of these went away in https://github.com/emscripten-core/emscripten/pull/17704/ but I don't see `aio.h` or `sys/sysctl.h`. Were they removed in a separate PR?

Reply:
This was removed via PR https://github.com/emscripten-core/emscripten/pull/18058. Hence, I've split this removal into commit 476bf225658143f5689fa2986e6fb491c8081eaf.

Reply:
Oh, indeed `sys/sysctl.h` has been removed via https://github.com/emscripten-core/emscripten/pull/18863. Let me split that change as well.

Reply:
Fixed with commit fbeecf3e812b05a4357d0e2ef663766f8cbe2b2b.